build.gradle

```diff
@@ -33,7 +33,7 @@ subprojects {
 //                'edu.washington.cs.types.checker:checker-framework:+',
                 'org.openjdk.jmh:jmh-generator-annprocess:+'
 
-        testCompile 'org.junit.jupiter:junit-jupiter-api:+',
+        testCompile 'org.junit.jupiter:junit-jupiter-api:5.0.0-M4',
```

Comment:
I wasn't able to run tests with latest ("+") junit version... Seems like we discussed that bug in latest junit build week ago or so?..